app/code/Magento/Backend/App/Response/Http/FileFactory.php

```diff
@@ -96,17 +96,51 @@ protected function _redirect($path, $arguments = [])
      * @param string $contentType
      * @param int $contentLength    explicit content length, if strlen($content) isn't applicable
      * @return \Magento\Framework\App\ResponseInterface
+     * @throws \Exception
+     *
+     * @deprecated
```

Comment:
Please mention the reason of deprecation according to the guide https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#php-js-and-xml